website/app/infrastructure/session/create-session-manager.server.ts

```diff
@@ -0,0 +1,79 @@
+import { createCookieSessionStorage } from '@remix-run/node';
```

Comment:
Why is session management infrastructure? 🤔

Reply:
again the create is infrastructure, the contract is not.
it relies on Remix, therefore it's infra to me